GTG/gtk/data/task_editor.ui

```diff
@@ -33,7 +33,7 @@
             <property name="receives_default">False</property>
             <property name="tooltip_text" translatable="yes">Mark this task as to be done</property>
             <property name="action_name">app.dismiss</property>
-            <property name="text" translatable="yes">Undismiss Task</property>
+            <property name="text" translatable="yes">Undismiss</property>
```

Comment:
I don't think we should make this shorter for translating purposes. Only if it makes sense for the dialog conceptually.